portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,4 +91,21 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /** Returns the integer entered by user on how many comments  */
+  private int getMaxComments(HttpServletRequest request) {
```

Comment:
I think getMaxComments might be a little misleading for the name of this function, since the user might request fewer than the maximum number of comments in the datastore. Maybe getNumDisplayComments or something similar?